streams/src/test/java/org/apache/kafka/streams/integration/InternalTopicIntegrationTest.java

```diff
@@ -109,53 +113,53 @@ private boolean isUsingCompactionForStateChangelogTopics() {
 
     @Test
     public void shouldCompactTopicsForStateChangelogs() throws Exception {
-        List<String> inputValues = Arrays.asList("hello", "world", "world", "hello world");
+        final List<String> inputValues = Arrays.asList("hello", "world", "world", "hello world");
 
         //
         // Step 1: Configure and start a simple word count topology
         //
         final Serde<String> stringSerde = Serdes.String();
         final Serde<Long> longSerde = Serdes.Long();
 
-        Properties streamsConfiguration = new Properties();
+        final Properties streamsConfiguration = new Properties();
         streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "compact-topics-integration-test");
         streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers());
         streamsConfiguration.put(StreamsConfig.ZOOKEEPER_CONNECT_CONFIG, CLUSTER.zKConnectString());
         streamsConfiguration.put(StreamsConfig.KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         streamsConfiguration.put(StreamsConfig.VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getPath());
         streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-        KStreamBuilder builder = new KStreamBuilder();
+        final KStreamBuilder builder = new KStreamBuilder();
 
-        KStream<String, String> textLines = builder.stream(DEFAULT_INPUT_TOPIC);
+        final KStream<String, String> textLines = builder.stream(DEFAULT_INPUT_TOPIC);
```

Comment:
Personally i feel adding `final` to any locals etc that don't change is good practice. It makes it immediately clear that the value of the field/local is fixed.